### ISSUE TITLE
Missing mapping definition

### ISSUE THREAD

**Issue by nilsga** [markkimsal/statsd-elasticsearch-backend#14]
When running this backend with StatsD, I get a type in the index which is not defined in the `es-index-template.sh` script:

```
"timer_data_stats": {
            "properties": {
               "@timestamp": {
                  "type": "string"
               },
               "act": {
                  "type": "string"
               },
               "count": {
                  "type": "string"
               },
               "count_90": {
                  "type": "string"
               },
               "count_ps": {
                  "type": "string"
               },
               "grp": {
                  "type": "string"
               },
               "lower": {
                  "type": "string"
               },
               "mean": {
                  "type": "string"
               },
               "mean_90": {
                  "type": "string"
               },
               "median": {
                  "type": "string"
               },
               "ns": {
                  "type": "string"
               },
               "std": {
                  "type": "string"
               },
               "sum": {
                  "type": "string"
               },
               "sum_90": {
                  "type": "string"
               },
               "sum_squares": {
                  "type": "string"
               },
               "sum_squares_90": {
                  "type": "string"
               },
               "tgt": {
                  "type": "string"
               },
               "upper": {
                  "type": "string"
               },
               "upper_90": {
                  "type": "string"
               }
            }
         }
```

This causes the mapping for this type to be dynamically created, and the `@timestamp` field becomes `string` instead of `date`.

**Comment by nilsga** [comment]
Not sure if it is related, but this code line seems suspicious: https://github.com/markkimsal/statsd-elasticsearch-backend/blob/master/lib/elasticsearch.js#L223-L224

Looks to me like the `elasticTimerType` possibly can be overwritten by another config value.

**Comment by markkimsal** [comment]
You are correct, the timerDataType was never configurable, it was always set to the timerType + '_stats'.  This is going to be the default value for timerDataType config value (timerType + '+stats').

**Comment by markkimsal** [comment]
I fixed this and released 0.3.0 because most people will have a config value of "timer_data" and the system was saving to "timer_data_stats".  I forgot to mention this issue number in the commit log.
